documentsManager/tests.py

```diff
@@ -747,6 +835,54 @@ def test_create_a_model(self):
         self.assertTrue(isinstance(evaluator_event, EvaluatorEvent))
 
 
+class EvaluatorEventTest(TestBase):
+    def test_accept_invitation_state(self):
+        event = self.create_event()
+        evaluator = self.create_evaluator()
+        evaluator_event = EvaluatorEvent.objects.create(
```

Comment:
Remember ident should be:
`valuator_event = EvaluatorEvent.objects.create(
            event=event,
            evaluator=evaluator
)`

Reply:
I think the linter of VSC is doing this automatically.
Fixin right now.
ty for the review :D